tests/unit/browser/webkit/network/test_httpproxy.py

```diff
@@ -0,0 +1,38 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2015-2016 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for browser.webkit.network.httpproxy."""
+
+from PyQt5.QtCore import QUrl
+
+from qutebrowser.browser.webkit.network.httpproxy import HTTPProxyResolver
+
+
+PAC_URL = "https://antizapret.prostovpn.org/proxy.pac"
```

Comment:
I'd like to avoid loading stuff from the internet in tests - this should use the local HTTP server which is already used for end-to-end tests, though it might take me a while until I can look into how to use that for unit tests too - or maybe you want to look into that?